Dockerfile.template

```diff
@@ -51,6 +52,8 @@ RUN set -ex; \
 		libjpeg-turbo-dev \
 		libpng-dev \
 		libwebp-dev \
+		libheif-dev \
```

Comment:
As noted in my previous comment, I think we can make this clear enough by moving this up to the `		imagemagick-dev \` line like this:

```dockerfile
		imagemagick-dev libheif-dev \
```

(so that they're clearly related)